activerecord/test/cases/relations_test.rb

```diff
@@ -1911,7 +1918,7 @@ def test_presence
     authors_count = Post.select("author_id, COUNT(author_id) AS num_posts").
       group("author_id").order("author_id").includes(:author).to_a
 
-    assert_no_queries do
+    assert_queries(1) do
```

Comment:
I wonder if this and all other examples original query should also be wrapped in an `assert_queries(1)` block to ensure there are no cases where unnecessary queries are happening